.github/workflows/export-and-release.yml

```
@@ -164,3 +164,40 @@ jobs:
           generateReleaseNotes: true
           tag: ${{ github.ref_name }}
           artifacts: ${{ steps.export.outputs.archive_directory }}/*
+
+      # Deploy the dedicated server build to the official game server droplet.
+      # Skipped gracefully if the deploy secrets are absent.
+      - name: Deploy dedicated server
+        continue-on-error: true
+        if: ${{ secrets.DROPLET_HOST != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)